docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -293,7 +293,7 @@ The output would be similar to the following:
                 "level": "info",
                 "value": 2048
             },
-...
+…
```

Comment:
```suggestion
...
```

Reply:
Reverted it. I thought these three dots are causing GitHub to highlight half of the page as red. That's not the case, don't know why it happens with some pages.